ui/package-lock.json

```diff
@@ -1,19637 +1,8 @@
 {
   "name": "ui",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
   "requires": true,
-  "packages": {
```

Comment:
Something happened with your Node installation/configuration. It seems it changed the version of the `package-lock.json` files. Please undo all of the changes to these files and keep them unchanged for now. Be sure you're using Node `12.x` and npm `6.x`.